app/Platform/RouteServiceProvider.php

```diff
@@ -60,13 +69,16 @@ protected function mapWebRoutes(): void
             // Route::get('game/{game}/players', [GamePlayerController::class, 'index'])->name('game.player.index');
             Route::get('game/{game}/dev-interest', GameDevInterestController::class)->name('game.dev-interest');
 
+            // Route::get('game/{game}/hashes', [GameHashController::class, 'index'])->name('game.hashes'); "Supported Game Files"
+            Route::get('game/{game}/hashes/manage', [GameHashController::class, 'manage'])->name('game.hashes.manage');
```

Comment:
Management pages require an authenticated user.
To make sure users are prompted to log in and get redirected to this route after the fact I think this (and the CRUD routes below) should be moved into the auth middleware block below.
It needs adjustment though because the `verified` middleware won't work just yet (I'm working on migrating our custom email verification to framework defaults):
```php
Route::group([
    'middleware' => ['auth'], // TODO 'verified'
], function () {
    Route::get('game/{game}/hashes/manage', [GameHashController::class, 'manage'])->name('game.hashes.manage');
    Route::resource('game-hash', GameHashController::class)->parameters(['game-hash' => 'gameMd5Hash'])->only(['update', 'destroy']);
```

Reply:
Makes sense - adjusted in latest.